lib/ts/BufferWriter.h

```diff
@@ -652,6 +652,22 @@ bwformat(BufferWriter &w, BWFSpec const &, char c)
   return w.write(c);
 }
 
+inline BufferWriter &
+bwformat(BufferWriter &w, BWFSpec const &spec, bool f)
+{
+  static const ts::string_view txt_true{"true"};
```

Comment:
Unfortunately not in C++11. There is a fix in C++14 that makes `strlen` able to be `constexpr` for literal strings, but currently `strlen` prevents `constexpr`.

The `txt` is to avoid compiler errors if it were named `true` and `false`.